apps/web/src/server/lib/multer.js

```diff
@@ -1,3 +1,8 @@
 import multer from 'multer';
+import path from 'path';
+import url from 'url';
 
-export const storage = multer.memoryStorage();
+const TMP_DIR = path.join(url.fileURLToPath(new URL('.', import.meta.url)), '../../../.tmp');
```

Comment:
I think we need to gitignore this folder. And if you don't mind we should rename it to `.temp`.